xcodeproj/internal/xcode_targets.bzl

```
@@ -89,12 +90,20 @@ def _make_xcode_target(
     Returns:
         A mostly opaque `struct` that can be passed to `xcode_targets.to_dto`.
     """
+    compile_target_present = (compile_target_swift or compile_target_other)
```

Multi-part suggested changes, because GitHub UI is limiting...

1:
```suggestion
    compile_targets= []
    if compile_target_swift:
        compile_targets.append(compile_target_swift)
    if compile_target_other:
        compile_targets.append(compile_target_other)
```